jetty-core/jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -435,11 +435,27 @@ public void initializeDefaults(Object object)
      * @return String to resolved and normalized path, or null if dir or destPath is empty.
      */
     public static String resolvePath(String dir, String destPath)
+    {
+        Path resolved = resolvedPath(dir, destPath);
+        if (resolved == null)
+            return null;
+        return resolved.toString();
+    }
+
+    /**
+     * Utility method to resolve a provided path against a directory.
+     *
+     * @param dir the directory (should be a directory reference, does not have to exist)
+     * @param destPath the destination path (can be relative or absolute, syntax depends on OS + FileSystem in use,
+     * and does not need to exist)
+     * @return String to resolved and normalized path, or null if dir or destPath is empty.
+     */
+    public static Path resolvedPath(String dir, String destPath)
```

Comment:
That's a confusing name, how is this method different from `resolvePath()`?

Reply:
The return value is the main difference.

One is a String, the other is a Path.

Reply:
Lets make naming this better a followup PR.

Reply:
Once a ting is written... im fo sémpere!